spring-native/README.md

```diff
@@ -1,17 +1,30 @@
 # OpenTelemetry Spring Native Example
 
-This example demonstrates usage of the [OpenTelemetry Spring starter](https://github.com/open-telemetry/opentelemetry-java-instrumentation/tree/main/instrumentation/spring/starters/spring-boot-starter) with a Spring Boot application and a GraalVM native image.
+This example demonstrates usage of the [OpenTelemetry Spring starter](https://opentelemetry.io/docs/instrumentation/java/automatic/spring-boot/) with a Spring Boot application running on a GraalVM native image.
 
-It consists of a Spring Boot application with:
+The example uses the following elements:
 
-- A simple web API available at `GET http://localhost:8080/ping`. When called,
-  the OpenTelemetry Spring starter records spans and metrics.
+- A web API available at `GET http://localhost:8080/ping`.
 - A docker compose setup configured to run the application and export to
   the [collector](https://opentelemetry.io/docs/collector/) via OTLP.
-- The collector is configured with
+- A collector configured with
   the [OTLP receiver](https://github.com/open-telemetry/opentelemetry-collector/tree/main/receiver/otlpreceiver)
-  and export it to standard out with
-  the [logging exporter](https://github.com/open-telemetry/opentelemetry-collector/tree/main/exporter/loggingexporter)
+  and exporting to the standard output with
+  the [logging exporter](https://github.com/open-telemetry/opentelemetry-collector/tree/main/exporter/loggingexporter).
+
+# Description of the instrumentation set-up
+
+We have included the [OpenTelemetry Spring starter](https://opentelemetry.io/docs/instrumentation/java/automatic/spring-boot/) to instrument the HTTP calls and send the OpenTelemetry data via OTLP:
+
+```kotlin
+ implementation("io.opentelemetry.instrumentation:opentelemetry-spring-boot-starter")
```

Comment:
where is the version?

Reply:
The version is defined for all the project examples in the https://github.com/open-telemetry/opentelemetry-java-examples/blob/main/build.gradle.kts file.